app/main.py

```diff
@@ -1,7 +1,24 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import requests
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+
+def get_weather() -> dict:
+    URL = "http://api.weatherapi.com/v1/current.json"
+    CITY = "Paris"
+    res = requests.get(f'{URL}?key={os.getenv("API_KEY")}&q={CITY}')
```

Comment:
It's a good practice to check the response status code before attempting to parse the JSON. Consider adding a check like `if res.status_code == 200:` to ensure the request was successful.